Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -99,6 +99,15 @@ public static async Task<bool> BuildAppxAsync(UwpBuildInfo buildInfo, Cancellati
                 return false;
             }
 
+            // Need to add ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch to MixedRealityToolkit.vcxproj
```

Comment:
I *think* I saw that this was fixed in the latest VS 2019

https://github.com/microsoft/msbuild/issues/3746